### PR TITLE
DocStringParser: Fix error message for empty parameter description.

### LINKED ISSUE #3728: Docstring parser cannot cope with empty description for parameter
```
/**
     *  @notice  vote to somebody
     *  @param {uint} id
     *  @return uint votes of candidates[id]
     */
    function vote(uint id) public returns (uint) {
        require(id >= 1 && id <= candidates.length);
        candidates[id].votes++;
        voteTo[msg.sender] = id;
        return candidates[id].votes;
    }
```
DocstringParsingError: Documented parameter "id
" not found in the parameter list of the function.

Why the compiler can not find parameter id?

### PR DESCRIPTION
**Pull request by ekpyron**
Fixes #3728.

### CODE REVIEW

File: libsolidity/parsing/DocStringParser.cpp [hunk 1]
```diff
@@ -119,21 +119,17 @@ DocStringParser::iter DocStringParser::parseDocTagParam(iter _pos, iter _end)
...
-	if (nameEndPos == _end)
-	{
-		appendError("End of param name not found: " + string(nameStartPos, _end));
-		return _end;
-	}
 	auto paramName = string(nameStartPos, nameEndPos);
 
 	auto descStartPos = skipWhitespace(nameEndPos, _end);
-	if (descStartPos == _end)
+	auto nlPos = find(descStartPos, _end, '\n');
```
Comment (chriseth) [inline libsolidity/parsing/DocStringParser.cpp:125]: Shouldn't this be `find(nameEndPos, _end, '\n')`?

**Review by chriseth — CHANGES_REQUESTED** [review]
Please add a changelog entry and a test case.

File: libsolidity/parsing/DocStringParser.cpp (hunk 1 above)
Comment (ekpyron) [reply libsolidity/parsing/DocStringParser.cpp:125]: nlPos is the end of the description, so searching starts at the beginning of the description... descStartPos might be _end already, but that's ok - then nlPos will be _end as well and the error about a missing description will be triggered...

**Comment by ekpyron** [comment]
@chriseth Added Changelog entry and test case.

**Review by chriseth — APPROVED** [review]

**Pull request merged**